### PR TITLE
Crowdin monthly November 2020

### PR DESCRIPTION
**Pull request by Julievrz**
Publish Crowdin monthly November 2020

### CODE REVIEW

File: news/_posts/translation/do-you-speak-prestashop-nov-2020.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+date:  2020-12-08 17:00:00
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it is also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in November!
+
+[Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
```
Comment (LouiseBonnard) [inline news/_posts/translation/do-you-speak-prestashop-nov-2020.md]:
```suggestion
![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
```

File: news/_posts/translation/do-you-speak-prestashop-nov-2020.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## Project news
+
+
+### The Hebrew project is progressing!
+
+In just a month, the Hebrew project went from 64% to 83% translated and validated, an increase of 19%! What a great evolution! :scream: Congratulations! Special thanks to  [danielshapiro9](https://crowdin.com/profile/danielshapiro9) who has joined us just about a month ago and who has already translated 26,091 words. Thank you for your involvement! And of course, thank you to the other translators of the Hebrew project!
```
Comment (LouiseBonnard) [inline news/_posts/translation/do-you-speak-prestashop-nov-2020.md]: Hey, the first sentence needs to be updated as the graph in ## Languages with the best evolution mentions an increase of 20%. :-)

Comment (LouiseBonnard) [reply news/_posts/translation/do-you-speak-prestashop-nov-2020.md]:
```suggestion
In just a month, the Hebrew project went from 64% to 83% translated and validated, an increase of 19%! What a great evolution! :scream: Congratulations! Special thanks to [danielshapiro9](https://crowdin.com/profile/danielshapiro9) who has joined us just about a month ago and who has already translated 26,091 words. Thank you for your involvement! And of course, thank you to the other translators of the Hebrew project!
```

**Comment by LouiseBonnard** [comment]
Don't forget to add the date of publication in the URL. 🙏

File: news/_posts/translation/2020-12-09-do-you-speak-prestashop-nov-2020.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+Contributing to PrestaShop is not only about the code, it is also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in November!
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### The Hebrew project is progressing!
+
+In just a month, the Hebrew project went from 63% to 83% translated and validated, an increase of 20%! What a great evolution! :scream: Congratulations! Special thanks to  [danielshapiro9](https://crowdin.com/profile/danielshapiro9) who has joined us just about a month ago and who has already translated 26,091 words. Thank you for your involvement! And of course, thank you to the other translators of the Hebrew project!
```
Comment (LouiseBonnard) [inline news/_posts/translation/2020-12-09-do-you-speak-prestashop-nov-2020.md]:
```suggestion
In just a month, the Hebrew project went from 63% to 83% translated and validated, an increase of 20%! What a great evolution! :scream: Congratulations! Special thanks to [danielshapiro9](https://crowdin.com/profile/danielshapiro9) who has joined us just about a month ago and who has already translated 26,091 words. Thank you for your involvement! And of course, thank you to the other translators of the Hebrew project!
```

File: news/_posts/translation/2020-12-09-do-you-speak-prestashop-november-2020.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+| 5. | Benjamin Gantikow (bbbenjie) | German | 778
+| 6. | SeongHyeon Cho (jaymz9634) | Korean | 602
+| 7. | Sretko Devič (Chico) | Slovenian | 429
+| 8. | Adeko Webdesign & Development (Adeko) | Dutch | 420
+| 9. | Cha (cafetango) | Chinese Traditional | 393
+| 10. | Marcin Orzechowski (Martinovy) | Polish | 210
+ 
+Congrats, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who's been contributing to our translation project thanks to the [Translators page](http://translators.prestashop.com/).
```
Comment (matks) [inline news/_posts/translation/2020-12-09-do-you-speak-prestashop-november-2020.md]:
```suggestion
Remember, you can see who's been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
```

More secure 😉

File: news/_posts/translation/2020-12-09-do-you-speak-prestashop-november-2020.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+ 
+## Complete translations
+ 
+#### Fully translated languages
+ 
+At the end of November, PrestaShop 1.7.7 was fully available (= 100% translated and validated) in **5 languages**:
+ 
+| **Chinese traditional** | **Danish** | English | Portuguese, Brazil |
```
Comment (matks) [inline news/_posts/translation/2020-12-09-do-you-speak-prestashop-november-2020.md]: Did you want to make a table? Then it must be on a single line.

Comment (Julievrz) [reply news/_posts/translation/2020-12-09-do-you-speak-prestashop-november-2020.md]: Yes, thank you!

**Review by matks — APPROVED** [review]

**Pull request merged**